lib/bokken/application.ex

```diff
@@ -15,9 +15,11 @@ defmodule Bokken.Application do
       # Start the PubSub system
       {Phoenix.PubSub, name: Bokken.PubSub},
       # Start the Endpoint (http/https)
-      BokkenWeb.Endpoint
+      BokkenWeb.Endpoint,
       # Start a worker by calling: Bokken.Worker.start_link(arg)
       # {Bokken.Worker, arg}
+      Bokken.Periodically
```

Comment:
I think this should be called `Bokken.BirthdayNotifier`. The current_name implies that it is a generic scheduler that can run arbitrary tasks.